CONTRIBUTING.md

```diff
@@ -98,9 +98,9 @@ vcpkg install ensmallen:x64-windows
 
 ### Build and Test
 
-This section describes how to build the **ensmallen** tests from source. **ensmallen** uses CMake as its build system and [Catch2](https://github.com/catchorg/Catch2) as the unit test framework.
+This section describes how to build the **ensmallen** tests from the source. **ensmallen** uses CMake as its build system and [Catch2](https://github.com/catchorg/Catch2) as the unit test framework.
```

Comment:
```suggestion
This section describes how to build the **ensmallen** tests from source. **ensmallen** uses CMake as its build system and [Catch2](https://github.com/catchorg/Catch2) as the unit test framework.
```

I would prefer to go with "from source", unless you like to write "from the source code", but "from source" is more common.